### DISCUSSION TITLE
Can any user defined markers be used as milestones?

### DISCUSSION THREAD

**Discussion by kavitharaju** [usfm-bible/tcdocs#35] [General]
In some of the usfmjsTests, `\k-s` and `\k-e` markers are used. Can any user defined names be milestones? Or are they limted to the predefined ones(`qt-s` and `ts-s`) and those starting with `z-`?
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/57-TIT.greek.oldformat/origin.usfm#L19
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/57-TIT.greek/origin.usfm#L73
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/samples-from-wild/UGNT2/origin.usfm#L91
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/samples-from-wild/UGNT1/origin.usfm#L24
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/45-ACT.ugnt.oldformat/origin.usfm

**Comment by mhosken** [comment]
There are some unclosed milestones here. For example, the first example on line 19 needs a final \* as in:

```
\k-s|x-tw="rc://*/tw/dict/bible/kt/jesus"\*
```
Notice also that after a | you can only have attributes and not general text or other markers. Not that you can have both attributes and sub markers at all anyway.
